packer/studio_virtualbox.pkr.hcl

```diff
@@ -0,0 +1,179 @@
+packer {
+  required_version = ">= 1.7.0, < 2.0.0"
+
+  required_plugins {
+    virtualbox = {
+      source  = "github.com/hashicorp/virtualbox"
+      version = ">= 1.0.0, < 2.0.0"
+    }
+  }
+}
+
+variables {
+  cpus              = 4
+  memory            = 8192
+  disk_size         = "50000"
+  headless          = false
+  iso_path_external = "https://releases.ubuntu.com/releases/jammy"
+  iso_file          = "ubuntu-22.04.2-live-server-amd64.iso"
+  iso_checksum      = "sha256:5e38b55d57d94ff029719342357325ed3bda38fa80054f9330dc789cd2d43931"
```

Comment:
During the time, this will require to update the `iso_file` and checksum. Ubuntu makes updates of the images periodically. The old images are available under 
https://old-releases.ubuntu.com/releases/jammy/